tests/builtins/test_all.py

```diff
@@ -23,6 +23,29 @@ def test_all_typeerror(self):
                 print("Done.")
         """)
 
+    def test_all_doc(self):
+        self.assertCodeExecution("""
+            print(all.__doc__)
+        """)
```

Comment:
Not necessary for this PR - but couldn't this test be an automated piece of the Builtins test class?

Reply:
Good call. I'll put it on my list. Probably a lot of ones to update to the new versions.